src/ml/neural_net/image_augmentation.hpp

```diff
@@ -223,6 +223,13 @@ class image_augmenter {
      * -0.05 * pi and 0.05 * pi.
      */
     float hue_max_jitter = 0.f;
+
+    /**
+     * When set to true, the images will only be resized in the output shape
+     * Used by the subclass of float_array_image_augmenter, to only resize
+     * images in case of prediction and evaluation.
+     */
+    bool resize_only = false;
```

Comment:
Why is this needed? The default values of this struct imply resize-only anyway. See https://github.com/apple/turicreate/blob/108139b62f77a2965de4c858ec21290e30e8123e/src/toolkits/object_detection/object_detector.cpp#L576

Reply:
So I can just check if the default settings are on and I should just pass resize flag to python based on the check.